docs/docs/cmd/teams/chat/chat-list.md

```diff
@@ -10,9 +10,15 @@ m365 teams chat list [options]
 
 ## Options
 
-`-t, --type [chatType]`
+`-t, --type [type]`
 : The chat type to optionally filter chat conversations by type. The value can be `oneOnOne`, `group` or `meeting`.
 
+`--userId [userId]`
+: ID of the user. Has to be specified when using application permissions. Specify either `userId` or `userName`, but not both.
+
+`--userName [userName]`
+: UPN of the user. Has to be specified when using application permissions. Specify either `userId` or `userName`, but not both.
+
 --8<-- "docs/cmd/_global.md"
 
 ## Examples
```

Comment:
Could you please revisit the examples? I think there was a copy-paste error

Reply:
Correct, my apologies.